### ISSUE TITLE
Support single quote JS strings

### ISSUE THREAD

**Issue by bhollis** [magnars/string-edit.el#7]
First, thanks for such a cool package! This should solve a problem I've run into many times with editing HTML inside JS.

One issue I've noticed, though, is exemplified by the following example:

```js
var foo = 'foo' +
  'bar';
```

If I run `string-edit-at-point` within either `'foo'` or `'bar'` in a buffer with the `js2-mode` major mode, the `*string-edit*` buffer shows only the string that I have the point on, not both strings. Furthermore, when I commit the change with `C-c C-c`, the string is changed to double quotes. It would be nice if this worked with single-quoted strings and preserved their quote character.

**Comment by bhollis** [comment]
It might also be nice to support strings like this:

```javascript
var result = [
        '<div class="container">',
        '  <div class="row">',
        '  </div>',
        '</div>'
      ].join('');
```

**Comment by magnars** [comment]
Agreed on both accounts. I don't much use JavaScript these days, but a pull request or two would certainly be welcome.

**Comment by gopar** [comment]
Python strings can also be single or double. Package converts single quote strings to double.

**Comment by Fuco1** [comment]
It would be also cool to leave back-tick escaped javscript in backticks (the "template strings")

example

```js
var rawHtml = `<row>
    <label alignment="left" class="text-dark font-weight-normal">Foo</label>
</row>`;
```

**Comment by Fuco1** [comment]
@magnars Can you assign me to this issue, I will try to get this done.

**Comment by Yevgnen** [comment]
Any news on this?

**Comment by magnars** [comment]
@Fuco1 I have added you as a collab to this repo, if you want to take a look.
